src/FileUtility.java

```diff
@@ -9,15 +9,15 @@
 public class FileUtility extends HoursReport {
 
 	static JFileChooser fileChooser;
-	static ArrayList<String> projects = new ArrayList<>();
-	static ArrayList<Date> dates = new ArrayList<>();
-	static ArrayList<String> names = new ArrayList<>();
-	static ArrayList<Double> duration = new ArrayList<>();
+	static ArrayList<Object> projects = new ArrayList<>();
```

Comment:
I was planning on using them later, when I have to fill in the sheet with data in the necessary column

Reply:
Also, there isn't an `ArrayObject` in the API - Only `ArrayList<Object>` (that I found) - maybe @aadupirn can confirm or deny? - So far, I need it to be object since `Cell` is considered an object

Reply:
Sorry, that was a typo. I mean `ArrayList<Cell>`, not `ArrayObject<Cell>`.

Casting a `Cell` to an `Object` loses important type information; namely, that it's a `Cell`. But, a `String` is also an `Object`. If I have `ArrayList<Object> projects`, I can add both a `Cell` and a `String` to the collection. But, a `Cell` and a `String` are very different things. If I accidentally call a `Cell` method on a `String`, I'm gonna have a bad time.

Do you need a collection of heterogeneous objects? Is the `projects` going to hold things other than `Cell`? If so, there are approaches we can take to work around that.

Reply:
Changed to `ArrayList<Cell>` I'll roll with that and see how that works out

Reply:
There's `Strings, Dates, and Doubles` in their respective array - however I did find that apache has a `dataFormatter` to change cells into the proper format as needed - going to try to add some loops in `ExcelWriter` to try and populate sheets